project-management/backlog-refinement-meetings/03-31-2020.md

```diff
@@ -0,0 +1,84 @@
+* Registers users and collects necessary demographic information, such as connection to autism, and is compliant with the UK Data Protection Act.
+* Takes input from users reporting their experiences navigating different sensory environments.
+* Securely transfers data from users categorised as usable for research and/or usable for public view, or neither, to the Open Humans backend file management system.
+* Allows moderation of input before it is made public.
+* Presents users with their own data and consent decisions.
+* Displays experiences that have been consented for public view to any visitor of the website.
+
+### Backlog Prioritisation: 
+
+* Fujitsu team to review and agree business requirements: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/94
+* Review and agree agile open source working process: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/97
+* Create sprint report and demo Video https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/116
+    * Create documentation on wireframes, sprint report (preparatory work) provide links on GitHub to where hosted - record of decisions made and processes done - co-working session Georgia and Chris to put everything on GitHub
+* Complete Sprint Meetings Documentation: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/123
+* Document backlog definitions (user stories and goals): https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/103
+* Link 1:1 user testing documentation to ways of working: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/115
+    Speak to community members to clarify structure they would like to be followed - document conversations - make this part of the     review of the workflow documentation pull requests (linked above)
+* Move closed work on wireframes and personas onto main GitHub repo
+* Drill down into business requirements to clarify acceptance criteria: speak to the community
+* (How): Define strategy for how will go about talking to community to gather info for acceptance criteria - process of speaking to community can happen at the same time - create in conjunction with the community so community driven 
+
+### GA Actions: 
+
+* Publish summaries on GitHub (put into markdown and incorporate SI's suggestions)
+* Publish notes from meetings with TA, JS, and EH on GitHub
+* Go through Repo and organise - pull requests, labels, issues, milestones
+* Write up meeting on 27th put into markdown 
+* Put Fujitsu team in touch with community members - 
+* Set up co-working session with CL 
+
+### Meeting Minutes
+
+* Focus on documentation, process, and defining acceptance criteria before development work begins
+* Full sprint not yet in progress - preparatory work has occured, but interative design not yet in place - but a full sprint report will be published 
```

Comment:
Can we add a date to this? Published at the end of sprint 2 or retrospectively for sprint 1? What about the video?